webapp/channels/src/utils/utils.tsx

```diff
@@ -1757,9 +1751,9 @@ const TrackFlowSources: Record<string, string> = {
 };
 
 function getTrackFlowSource() {
-    if (isMobile()) {
+    if (UserAgent.isMobile()) {
```

Comment:
Based on that discussion, it seems like we're actually the owners of this data since the reorg, so I'm going to say this works correctly. We also discussed just removing this entirely since we're not currently using it. This data seems to be used to track where people are generating invite links from and whether those links actually get used